doc/classes/AudioEffectLimiter.xml

```diff
@@ -68,12 +69,15 @@
 	</methods>
 	<members>
 		<member name="ceiling_db" type="float" setter="set_ceiling_db" getter="get_ceiling_db">
+			Limits the waveform to don't overpass this value. Value can range from -20 to -0.1. Default value: [code]-0.1dB[/code].
```

Comment:
This sentence is awkward.  Maybe: "The waveform's maximum allowed value."  or "peak allowed value"?